Sources/AppleAPI/Client.swift

```diff
@@ -27,6 +28,8 @@ public class Client {
                 return "Not a valid phone number index. Expecting a whole number between \(min)-\(max), but was given \(given ?? "nothing")."
             case .noTrustedPhoneNumbers:
                 return "Your account doesn't have any trusted phone numbers, but they're required for two-factor authentication. See https://support.apple.com/en-ca/HT204915."
+            case .notAuthenticated:
+                return "No authenticated session found"
```

Comment:
![image](https://user-images.githubusercontent.com/1119565/111545922-dd953080-8744-11eb-8f8b-3850c6842080.png)
Do you think the red is a little harsh for the error? It really just is an info statement. Perhaps the description here would be better a little less error like. `You are already signed out` ?

Reply:
I like both suggestions 👍 
Implemented it in d49c722